NFIQ2/NFIQ2Algorithm/src/features/FeatureFunctions.cpp

```diff
@@ -383,6 +383,7 @@ NFIQ2::QualityFeatures::getRidgeValleyStructure(const cv::Mat &blockCropped,
 	// platforms (10^10)
 
 	dt1.forEach<double>([&](double &val, const int *position) {
+		(void)position;
```

Comment:
No need to use it when you can remove the name.

```suggestion
	dt1.forEach<double>([&](double &val, const int *) {
```